cfme/tests/middleware/dballocator_methods.py

```diff
@@ -0,0 +1,96 @@
+import pytest
+from contextlib import closing
+from urllib2 import urlopen, HTTPError
+from cfme.middleware import parse_properties
+from utils import conf
+
+
+REQUESTEE = "cfme_tests"
+DS_UUID_PROP = "uuid"
+DB2_105 = 'db2_105'
+MSSQL_2014 = 'mssql2014'
+MYSQL_57 = 'mysql57'
+POSTGRESPLUS_94 = 'postgresplus94'
+POSTGRESQL_94 = 'postgresql94'
+SYBASE_157 = 'sybase157'
+ORACLE_12C = 'oracle12c'
+ORACLE_12C_RAC = 'oracle12cRAC'
+SEP = '='
+
+
+def db_allocate(db_type, expiry=1):
+    """
+    Method for allocating database via DBAllocator tool by providing database type,
+    expiry time in minutes and the name of requestee.
+
+    It returns the result of DBAllocator allocate action in dictionary format,
+    which contains the connection properties to allocated database.
+    From returned properties, very important one is 'uuid',
+        which will be used for manually deallocating database.
+
+    Args:
+        db_type: type of the database which is registered in DBAllocator,
+            all possible types are in this class.
+        expiry: reservation duration in minutes after
+            which database will be automatically deallocated (optional).
+
+    Returns:
+        database connection properties as dictionary containing information:
+            db.password: password
```

Comment:
given the dotted/nested structure i wonder if we shouldn't have a simple object representing the data and having the deallocate method (in order to localize information and remove the need to know the structure on use)